project1/src/main/java/uk/co/cpsd/javaproject1/GoatChart.java

```diff
@@ -0,0 +1,47 @@
+package uk.co.cpsd.javaproject1;
+
+import org.jfree.chart.ChartFactory;
+import org.jfree.chart.ChartPanel;
+import org.jfree.chart.ChartUtils;
+import org.jfree.chart.JFreeChart;
+import org.jfree.data.category.DefaultCategoryDataset;
+
+import javax.swing.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+public class GoatChart extends JFrame {
+
+    public GoatChart(List<Integer> goatCounts, List<Integer> grassCounts) {
+
+        try {
+            setTitle("Population Over Time");
+        setSize(700, 450);
```

Comment:
It's worth having an auto-formatter that will sort out things like indents.  Most diff tools get at least slightly confused by indent changes (let's face it, it matters for Python) so if you can have consistent format when you commit it keeps things simpler